app/controllers/wpcc/your_details_controller.rb

```diff
@@ -48,6 +51,12 @@ def session_params
 
     def amend_session
       your_details_form_params.each { |key, value| session[key] = value }
+      session[:manually_opt_in] = manually_opt_in?
+    end
+
+    def manually_opt_in?
```

Comment:
Renaming this will make it more understandable. 

`manually_opt_in?` --> `require_ manually_opt_in?`

This communicates the intention much clearer.